subsys/net/lib/http/http_server.c

```diff
@@ -566,6 +572,11 @@ static void http_received(struct net_app_ctx *app_ctx,
 
 	NET_DBG("[%p] Received %zd bytes http data", ctx, recv_len);
 
+	if (net_pkt_context(pkt)) {
+		ctx->http.parser.addr = &net_pkt_context(pkt)->remote;
```

Comment:
Does ctx->http.parser.addr ever get used anywhere?  I can't find it in this patch.

Reply:
It's used in websocket reply. https://github.com/zephyrproject-rtos/zephyr/pull/5035/commits/53eb1b6eab6d1f825c98e761b2e5995f767ed5a4#diff-ee0f87c7adaeba40cb43dd5749f9407aR354